roles/jenkins/templates/transfer-to-slaves-script

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# NOTE: each single whitespace and newline close to the template is important!
```

Comment:
Can you add a short description of this script's purpose? I.e. what it's meant to transfer.